Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -164,22 +164,23 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
               Real Gil_damp = PhysConst::mu0 * mag_gamma_interp
                               * MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(0,1,0),mag_alpha_arr)
                               / MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(0,1,0),mag_Ms_arr);
-              
+ 
               // x component on y-faces of grid
               M_yface(i, j, k, 0) += dt * (-PhysConst::mu0 * mag_gamma_interp) * ( M_yface(i, j, k, 1) * Hz_eff - M_yface(i, j, k, 2) * Hy_eff)
                 + dt * Gil_damp * ( M_yface(i, j, k, 1) * (M_yface(i, j, k, 0) * Hy_eff - M_yface(i, j, k, 1) * Hx_eff)
-                - M_yface(i, j, k, 2) * ( M_yface(i, j, k, 2) * Hz_eff - M_yface(i, j, k, 0) * Hz_eff));
-
+                - M_yface(i, j, k, 2) * ( M_yface(i, j, k, 2) * Hx_eff - M_yface(i, j, k, 0) * Hz_eff));
+	      
```

Comment:
white spaces.  i wonder why the white space checker isn't running?

Reply:
Because of the conflict file, I think. I suggest dealing with the white spacing later.

Reply:
sure.  it'll get caught when jackie's PR goes into reva's.